sites/hashai/resources/docs/simulation/creating-simulations/behaviors/inheritance.md

```diff
@@ -21,23 +21,23 @@ Consider Arnold Schwarzenegger, the indomitable bodybuilder-actor-poltician. _Is
 
-## Introducing Agent Types
+## Agent Types
 
 In HASH, agents are _in practice_ defined by their behaviors. While users are free to add a `type` field to their agents, and many simulation authors do so, these _types_ have until now been symbolic labels that have made tracking agents over the course of simulations easier, and have primarily been used to generate descriptive metrics and plots.
 
-We're now planning to introduce support for **Agent Types** in HASH which go further. In an upcoming update, hEngine will attempt to match the _type_ field of an agent, to a schema in hIndex.
+We're now planning to introduce support for **Agent Types** in a forthcoming update, which go further. In the future, [HASH Engine](/platform/engine) (hEngine) will attempt to match the _type_ field of an agent to an entity type in HASH, where it contains the URI of an _entity type_ schema.
 
-Schemas published to hIndex may describe either:
+Entity types created in HASH describe 'things'. In a simulation context these could for example be:
 
 1.  agents \(Agent Types\); or
 1.  actions taken by agents \(Behaviors\).
 
-Schemas consist of _properties_ with _expected types_, and in some cases these may have _default values_. Going forward, all agent schemas in HASH will contain a `behavior` property. This can be set to contain one or more behaviors.
+Entity types consist of _properties_, each of which has their _own_ property types, and in some cases these may have _default values_. Entity types in HASH may contain a `hash_simulation_behavior` property containing one or more [behaviors](/docs/simulation/creating-simulations/behaviors).
 
-When an agent in a HASH simulation has its _type_ declared going forward, if hEngine is able to match that type to an hIndex-published schema, any behaviors specified in that schema's `behavior` array will be attached to the agent.
+When an agent in a HASH simulation has its _type_ declared going forward, if hEngine is able to match that type to a valid entity type URI, any behaviors specified in that schema's `hash_simulation_behavior` array will be attached to the agent.
 
-There is no limit to the number of types that may be attached to an agent, and types may continue to be used descriptively without being linked to hIndex-listed schemas.
+There is no limit to the number of types that may be attached to an agent, and types may continue to be used descriptively without being linked to entity type URIs.
 
 <Hint style="info">
-As part of these changes, we'll be allowing users to publish their own schema definitions to hIndex, and attach arrays of hIndex-published behaviors to these schemas.
+As part of these changes, we'll be allowing users to attach arrays of HASH simulation behavior files to entity types under the protected `hash_simulation_behavior` property type field.
```

Comment:
I wonder if we want to use the term "list" instead of "array"? it sounds less technical.

Reply:
Implemented here: https://github.com/hashintel/hash/pull/1052/commits/775d9bd2f0bfe5ee0fc372edcef90f0daabfc31f

Reply:
I think that is better, thanks 👍